extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php

```diff
@@ -0,0 +1,20 @@
+<div id="article-video" class="article-video">
+	<img class="thumbnail" src="<?= $thumbnailUrl; ?>">
+	<div class="video-container">
+		<div class="video-placeholder">
+			<div class="video-thumbnail">
+				<img class="thumbnail" src="<?= $thumbnailUrl; ?>">
+				<img src="<?= $closeIconUrl; ?>" class="close">
+			</div>
+			<div class="video-details">
+				<div class="video-details-left">
+					<div class="video-time"><?= wfMessage( 'articlevideo-watch',
+							$videoDetails['time'] ); ?></div>
+					<div class="video-title"><?= $videoDetails['title']; ?></div>
```

Comment:
I know this data comes from safe source, but it's still possible that there are some characters that breaks HTML markup. For safety please use `htmlspecialchars()` and `Sanitizer::encodeAttribute()`